### PR TITLE
Calculate pull-request

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
-    return "";
+    std::string retunValue="";
+    std::map<std::string, int> mapValues= {{"add",0},{"subtract",1},{"multiply",2},{"divide",3}};
+
+    if(mapValues.find(command) == mapValues.end())
+    {
+        return retunValue="Invalid data";
+    }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:12]: You overcomplicated this.

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,38 @@
...
+            if(second == 0)
+            {
+                retunValue = "Division by 0";
+            }
+            else
+                retunValue = std::to_string( first / second);
+            break;
+       // default:
+         //   retunValue = "Invalid data";
+    }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:35]: Just use if

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+        case 3:
+            if(second == 0)
+            {
+                retunValue = "Division by 0";
+            }
+            else
+                retunValue = std::to_string( first / second);
+            break;
+       // default:
+         //   retunValue = "Invalid data";
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:34]: why???

**Review by nauka-programowania-MA — APPROVED** [review]

File: homework/calculate/calculate.hpp (hunk 3 above)
Comment (ms10335) [reply homework/calculate/calculate.hpp:34]: Hi,
I thought if I use "if " statement at the begining(line 9 -12), and check if command is not valid, function will never reach switch statemant and default case is not needed.

**Pull request closed without merging**